ui/components/component-library/header-base/header-base.tsx

```diff
@@ -70,7 +70,6 @@ export const HeaderBase: React.FC<HeaderBaseProps> = ({
     >
       {startAccessory && (
         <Box
-          className="mm-header-base__start-accessory"
```

Comment:
Removing classNames here as there are no associated styles and should encourage engineers to add their own className if required through `childrenWrapperProps` instead of using this one which could be subject to change.